Source/WebCore/Modules/identity/DigitalCredential.cpp

```diff
@@ -68,6 +68,11 @@ static ExceptionOr<UnvalidatedDigitalCredentialRequest> jsToCredentialRequest(co
     auto scope = DECLARE_THROW_SCOPE(document.globalObject()->vm());
     auto* globalObject = document.globalObject();
 
+    // Check that the object is JSON stringifiable.
+    ConversionResult<IDLJSON> conversionResult = JSC::JSONStringify(globalObject, request.data.get(), 0);
+    if (conversionResult.hasException(scope)) [[unlikely]]
+        return Exception { ExceptionCode::ExistingExceptionError };
```

Comment:
Instead of creating a `ConversionResult<IDLJSON>`, would it not suffice to check whether `scope` has an exception?


```suggestion
    auto jsonString = JSC::JSONStringify(globalObject, request.data.get(), 0);
    if (scope.exception()) [[unlikely]]
        return Exception { ExceptionCode::ExistingExceptionError };
```

Reply:
Yes, that could work... the reason I went for this approach was that is seemed more explicit about what exactly caused the error (via the ConversionResult).